src/features/stock-out.js

```diff
@@ -263,7 +263,15 @@ async function getStockOutConfigs({
       type: 'input',
       name: 'form_name',
       message: 'Enter stock out form ID',
-      default: 'stock_out'
+      default: 'stock_out',
+      when: function (answers){
```

Comment:
Could you update the readme with this? Or create another task to update the readme.

Reply:
I will create a ticket for updating the readme.

Reply:
I will create another ticket to update the readme.